### ISSUE TITLE
validation report and `use_vignettes`

### ISSUE THREAD

**Issue by mariev** [phuse-org/valtools#21]
it looks like `usethis::use_vignettes` automatically adds .gitignore to the vignettes folder which blocks .R and .html extensions. Need a workaround for this b/c validation reports for "validate on install" and "validate at distribution" may conflict.

Is whitelisting an option? seems like it should be: https://stackoverflow.com/questions/15288712/gitignore-whitelist-on-directory-and-its-contents

**Comment by thebioengineer** [comment]
This is a super interesting trick. I like it!

After some experimentation, in our `vt_use_validation_report()` (or whatever we call it) we add:

```
usethis::use_git_ignore(c("!*"), directory = "vignettes/validation")
```

**Comment by mariev** [comment]
closed via ae8e7b1
